crates/ignore/src/walk.rs

```diff
@@ -1699,6 +1693,8 @@ impl<'s> Worker<'s> {
                     // Wait for next `Work` or `Quit` message.
                     loop {
                         if let Some(v) = self.recv() {
+                            self.active_workers
+                                .fetch_add(1, AtomicOrdering::Relaxed);
```

Comment:
So I did like how previously there were methods with decentish names that made call sites a bit more descriptive. Maybe something like this:

```
// Inactivates a single worker and
// returns the total number of remaining
// active workers.
fn worker_inactivate(&mut self) -> bool { ... }

// Activates a single inactive worker.
fn worker_activate(&mut self) { ... }
```

Reply:
Good idea, done in d6f850f57436301fea602a5b4419ffacb464bcc9